src/test/java/marquez/db/DatasetDaoTest.java

```diff
@@ -221,8 +232,18 @@ public void testFindAll_limitOnly() {
 
   @Test
   public void testFindAll_offsetOnly() {
-    final List<DatasetRow> newDatasetRows =
-        newDatasetRowsWith(namespaceRow.getUuid(), datasourceRow.getUuid(), 4);
+    List<DatasetRow> newDatasetRows = new ArrayList<>();
+
+    Stream.<String>builder()
+        .add("a_test_dataset")
+        .add("b_test_dataset")
+        .add("c_test_dataset")
+        .add("d_test_dataset")
+        .build()
+        .forEach(
+            name ->
+                newDatasetRows.add(
+                    newDatasetRowWith(namespaceRow.getUuid(), datasourceRow.getUuid(), name)));
```

Comment:
Actually, I realized that naming (specifically the dataset name) is highly important to what you're looking to demonstrate in the test, which is why you opted to set them yourself :)

Reply:
Went with the sorting approach. Looks clean now.